token.rb

```diff
@@ -0,0 +1,18 @@
+require 'openssl'
```

Comment:
Add a block comment here, and point to source of the code. It's the gist mentioned in the mail.